ani-cli

```diff
@@ -377,7 +376,7 @@ quality=best
 scrape=query
 download_dir="."
 
-while getopts 'viq:dp:chDUV:' OPT; do
+while getopts 'viq:dp:chDUV' OPT; do
```

Comment:
I thought we needed that to catch the anime query, but turns out we don't. Nice catch.